docker/Dockerfile.exp

```diff
@@ -10,6 +10,7 @@ RUN conda update -n base -y conda && conda install -y -c conda-forge mamba && ma
 SHELL ["conda", "run", "--name", "esmvaltool", "/bin/bash", "-c"]
 
 COPY . /src/ESMValTool
+RUN mamba install -y -c conda-forge rasterio
```

Comment:
This is a bit suspect. Why isn't it coming from the environment.yml?

Reply:
This may be related to conda-forge/esmvalcore-feedstock#31

Reply:
ahaa! Good point, Klaus, let me remove and rerun the test see if it does the trick. No, it was picking it up from PyPi and was badly scratching my head as to why, didn't manage to find a solution, so I had to plop it in here as a kluge

Reply:
nope, same issue, `rasterio` being picked up from PyPi where it fails to build the wheels, have a look @zklaus @bouweandela

Reply:
nevermind that, I looked up the actual work done by the flow, and I see it installs the conda-forge ESMValCore `esmvalcore                     2.5.0rc3  pyh39db41b_0                 conda-forge/label/esmvalcore_rc/noarch` - so what's the point of this container anyway, since the conda install trumps the local install from the `latest` ESMValCore container

Reply:
fixed, see https://github.com/ESMValGroup/ESMValTool/pull/2551#issuecomment-1054452476 - bugger me flat, I went through the hoops with this